adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -1848,7 +1848,9 @@ abstract class AvroGenomicRDD[T <% IndexedRecord: Manifest, U <: Product, V <: A
     blockSize: Int = 128 * 1024 * 1024,
     pageSize: Int = 1 * 1024 * 1024,
     compressCodec: CompressionCodecName = CompressionCodecName.GZIP,
-    disableDictionaryEncoding: Boolean = false) {
+    disableDictionaryEncoding: Boolean = false,
+    enableHivePartitioning: Boolean = false,
+    hivePartitioningGenomicBinSize: Int = 1000000) {
```

Comment:
I would roll `enableHivePartitioning` and `hivePartitioningGenomicBinSize` into one parameter `optHivePartitioningBinSize`